### PR TITLE
feat(docs): improved main, standalone and widget introduction

### PR DESCRIPTION
**Pull request by zdrazil**
#### Changelog notes 

```
* Rephrased and restructured main, Distributor Widget and Distributor Standalone introduction.
* Added FAQ about configuration ids.
```

#### Check during review

- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

### CODE REVIEW

File: distributor-widget/README.md [hunk 1]
```diff
@@ -46,47 +76,80 @@ Mews.Distributor({
...
+        distributor.setEndDate(new Date(2018, 1, 3));
+    }
+);
 </script>
 ```
 
-To see a list of all available API calls, please consult [API ](reference.md#api-reference)section.
-
-Closing of Distributor is provided in the overlay by default, so you don’t have to worry about that.
+{% hint style="info" %}
```
Comment (zdrazil) [inline distributor-widget/README.md:144]: Should work like this: https://docs.gitbook.com/editing-content/markdown#hints-and-callouts

File: distributor-widget/README.md [hunk 2]
```diff
@@ -1,41 +1,71 @@
...
 ```text
 mews.li
 *.mews.li
 https://pay.datatrans.com/upp/payment/js/secure-fields-1.0.0.js
 ```
 
-The final datatrans url is for [PCI Proxy](https://www.pci-proxy.com/) which is the secure, PCI-DSS complaint solution that is used by our Merchant to process payment cards.
+The last, datatrans, URL is for [PCI Proxy](https://www.pci-proxy.com/) which is the secure, PCI-DSS complaint solution that is used by our Merchant to process payment cards.
+
+## Initialize Distributor Widget
```
Comment (zdrazil) [inline distributor-widget/README.md:55]: I forgot if `Distributor Widget` should be both uppercase (it's a special name) or not (generic name). Same for Distributor Standalone. What do you think? But let's not fix it in this PR, too keep it consistent with other sections of the guide and a changes a little smaller.

File: distributor-widget/README.md [hunk 3]
```diff
@@ -97,12 +160,3 @@ Mews.Distributor({
 });
 </script>
 ```
-
-## Styles <a id="styles"></a>
-
-Distributor does not use separate CSS files, everything is packed inside the script. For possible customizations, consult [Customization](reference.md) section.
```
Comment (zdrazil) [inline distributor-widget/README.md:103]: This doesn't fit anywhere even without referencing the non-existing customization section. So I've removed it. It's less confusing without it. If we want to have something like this, it should be rewritten somehow.

**Comment by zdrazil** [comment]
This PR just tries to improve the contents, it doesn't restructure anything in any major way, so it's easy to what changed and keep the number of changed lines smaller.

**Review by koprivajakub — APPROVED** [review]

**Pull request merged**